src/error.ts

```diff
@@ -65,8 +70,7 @@ export function createError(
 
   const err = new H3Error(
     input.message ?? input.statusMessage,
-    // @ts-ignore
-    input.cause ? { cause: input.cause } : undefined
```

Comment:
This is a v8 feature to provide event cause in traces ([read more](https://v8.dev/features/error-cause))